03/app.js

```diff
@@ -1,3 +1,12 @@
 const x = 10;
 let iteration = 0;
-let randomNumber = -1;
\ No newline at end of file
+let randomNumber = -1;
+
+
+while(randomNumber !== x) {
+    randomNumber = Math.round(Math.random() * 15);
+    iteration++;
+    console.log(randomNumber);
+}
+
+console.log('Losowanie odbyło się '+ iteration +' razy, zanim wylosowano liczbę 10 ');
```

Comment:
Tutaj zwróciłbym Twoją uwagę na to, że na końcu warto użyć zmiennej, aby zmiana w linii 1 nie wymuszała też zmiany w linii 12 czyli powinno być:
```
console.log('Losowanie odbyło się '+ iteration +' razy, zanim wylosowano liczbę ' + x); 
```